src/run.rs

```diff
@@ -652,8 +655,7 @@ fn vacuum(
             for repository_tag in &image_node.image_record.repository_tags {
                 if regex_set.is_match(&format!(
                     "{}:{}",
-                    repository_tag.repository,
-                    repository_tag.tag,
+                    repository_tag.repository, repository_tag.tag,
```

Comment:
Seems like there is a formatting lint issue here. I think it's because in CI we do some trickery to make `cargo fmt` aware of macro invocations, whereas generally `cargo fmt` ignores them. You can fix it manually or run `toast format` if you happen to have [Toast](https://github.com/stepchowfun/toast) installed.

Reply:
Nice tool, done :)